dev-aws/kafka-shared/iam_credentials.tf

```diff
@@ -21,3 +21,33 @@ module "iam_credentials_producer" {
 
   cert_common_name = "auth-customer/credentials-api"
 }
+
+
```

Comment:
Can you please use the consumer module?
Example: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/dev-aws/kafka-shared/customer-billing.tf#L23-L30

Reply:
https://github.com/utilitywarehouse/kafka-cluster-config/pull/72/commits/a8c6fcc5d4cbbd030be3daaf63947307abbff507